Documentation/api.md

```diff
@@ -394,6 +394,7 @@ PrometheusSpec is a specification of the desired behavior of the Prometheus clus
 | priorityClassName | Priority class assigned to the Pods | string | false |
 | portName | Port name used for the pods and governing service. This defaults to web | string | false |
 | arbitraryFSAccessThroughSMs | ArbitraryFSAccessThroughSMs configures whether configuration based on a service monitor can access arbitrary files on the file system of the Prometheus container e.g. bearer token files. | [ArbitraryFSAccessThroughSMsConfig](#arbitraryfsaccessthroughsmsconfig) | false |
+| honorTimestamps | HonorTimestamps configures Prometheus whether it should honor timestamps coming from metrics | *bool | false |
```

Comment:
This reads a bit strange to me. It implies we always set the `honor_timestamps` to false, which is not true because the default value is `true`.

WDYT of naming this `doNotHonorTimestamps` or something similar, then false will make sense to be the default value? Its an awful naming, but hopefully it gets the point across.

Reply:
Maybe we should implement the per servicemonitor/podmonitor setting in this PR and have this named as a parallel to #2806 ?